From a0567212372a4e1f131e4cfedd9b71714da7fe35 Mon Sep 17 00:00:00 2001
From: Chen Jun <chenjun102@huawei.com>
Date: Mon, 23 May 2022 21:18:33 +0800
Subject: [PATCH] perf: hisi: Fix unexpected modifications in
 hisi_uncore_l3c_pmu.c
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: a0567212372a4e1f131e4cfedd9b71714da7fe35
Modified-by-SEL: No


hulk inclusion
category: bugfix
bugzilla: https://gitee.com/openeuler/kernel/issues/I584X2
CVE: NA

--------------------------------

the name passed to cpuhp_setup_state_multi in hisi_uncore_l3c_pmu.c was
modification unexpected.

Fixes: 744d0990ad13 ("perf: hisi: Add support for HiSilicon SoC L3TPMU")
Signed-off-by: Chen Jun <chenjun102@huawei.com>
Reviewed-by: Weilong Chen <chenweilong@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 drivers/perf/hisilicon/hisi_uncore_l3c_pmu.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/perf/hisilicon/hisi_uncore_l3c_pmu.c b/drivers/perf/hisilicon/hisi_uncore_l3c_pmu.c
index 8b35f7cb4f38..2c71a8971723 100644
--- a/drivers/perf/hisilicon/hisi_uncore_l3c_pmu.c
+++ b/drivers/perf/hisilicon/hisi_uncore_l3c_pmu.c
@@ -608,7 +608,7 @@ static int __init hisi_l3c_pmu_module_init(void)
 	int ret;
 
 	ret = cpuhp_setup_state_multi(CPUHP_AP_PERF_ARM_HISI_L3_ONLINE,
-				      "AP_PERF_ARM_HISI_L3T_ONLINE",
+				      "AP_PERF_ARM_HISI_L3_ONLINE",
 				      hisi_uncore_pmu_online_cpu,
 				      hisi_uncore_pmu_offline_cpu);
 	if (ret) {
-- 
2.33.0

